chapter_multilayer-perceptrons/environment_vn.md

```diff
@@ -451,7 +463,9 @@ The confusion matrix C, is simply a $k \times k$ matrix where each column corres
 Each cell's value $c_{ij}$ is the fraction of predictions where the true label was $j$ *and* our model predicted $y$.
 -->
 
-*dịch đoạn phía trên*
+Để ước lượng phân phối của nhãn mục tiêu, đầu tiên ta dùng một bộ phân loại tương đối tốt (thường được học trên tập huấn luyện) và sử dụng một tập kiểm định (cùng phân phối với tập huấn luyện) để tính ma trận confusion.
```

Comment:
`confusion matrix` tạm block bởi #1022  
```suggestion
Để ước lượng phân phối nhãn mục tiêu, đầu tiên ta dùng một bộ phân loại sẵn có tương đối tốt (thường được học trên tập huấn luyện) và sử dụng một tập kiểm định (cùng phân phối với tập huấn luyện) để tính ma trận confusion.
```

Reply:
Đoạn này ban đầu mình cũng dịch `off the shelf` là `sẵn có` nhưng sợ sẽ gây confused cho người đọc vì có thể hiểu thành đây là một bộ phân loại đã có sẵn, tức không cần phải huấn luyện.